fc/fc_app.py

```diff
@@ -159,10 +169,12 @@ def _make_fc_tasks(index: Index,
     workflow = GridWorkflow(index, output_product.grid_spec)
 
     tiles_in = workflow.list_tiles(product=input_product.name, **query)
-    _LOG.info(f"{len(tiles_in)} {input_product.name} tiles in {repr(query)}")
+    #_LOG.info(f"{len(tiles_in)} {input_product.name} tiles in {repr(query)}")
+    _LOG.info('%d %s tiles in %s', len(tiles_in), input_product.name, str(repr(query)))
+    # _LOG.info('Found %d tasks', num_tasks_saved)
     tiles_out = workflow.list_tiles(product=output_product.name, **query)
-    _LOG.info(f"{len(tiles_out)} {output_product.name} tiles in {repr(query)}")
-
+    #_LOG.info(f"{len(tiles_out)} {output_product.name} tiles in {repr(query)}")
+    _LOG.info('%d %s tiles in %s', len(tiles_out), output_product.name, str(repr(query)))
```

Comment:
Why the change of logging method?

It used to be necessary to split logging string creation like this, but `f-strings` are now considered just as fast and completely acceptable.

Reply:
Travis was failing here. This was a fix to pass Travis at the time. I wasn't keen on the change either. The root issue with Travis is fixed now.